stretch-assignment/index.html

```diff
@@ -3,16 +3,31 @@
   <head>
     <title>DOM II</title>
     <link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.0.2/TweenMax.min.js"></script>
+
   </head>
   <body>
     <script defer="defer" src="./index.js"></script>
-
-    <div class="blocks">
-      <div class="block block--red"></div>
-      <div class="block block--blue"></div>
-      <div class="block block--green"></div>
-      <div class="block block--pink"></div>
-      <div class="block block--gray"></div>
+    <h1>Rocket Nyan Cat Game</h1>
```

Comment:
This was fun w the nyan cats flying off.

Reply:
thanks 😂 
Do you have any idea how to make it so the cats only are animated when the block is pressed?
like a jetpack kind of thing. I don't know how to make the event continuously fire. Looked into setInterval, but could not understand it via MDN and some youtube videos